src/Client.php

```diff
@@ -19,6 +19,8 @@
 
 class Client
 {
+    const API_CLIENT_VERSION = '1.21.0';
```

Comment:
I am not sure where `1.21.0` is coming from?

Instead of hard coding the version this would ideally use the installed version from Composer via `ocramius/package-versions`, `composer/package-versions-deprecated` or  `Composer\InstalledVersions`. Not 100% sure package to best rely on.

Reply:
I thought about hard coding that and updating it for each release.
How can I use `Composer\InstalledVersions`  or `ocramius/package-versions`  for the root package? These shows which packages are installed.

Reply:
Updating it with each release is something which can easily be missed.

I am not sure I understand the point about the root package? People install this package as a dependency in their projects. It will never be the root package

Reply:
what I'm trying to say is that doing something like this:

`$version = \PackageVersions\Versions::getVersion('packagist/private-packagist-api-client');`
 would throw an exception:
`Fatal error: Uncaught OutOfBoundsException: Package "packagist/private-packagist-api-client" is not installed `
and doing something like this would return the reference and not the release tag:
```$version = \Composer\InstalledVersions::getRootPackage();
array(5) {
  'pretty_version' =>
  string(10) "dev-master"
  'version' =>
  string(10) "dev-master"
  'aliases' =>
  array(0) {
  }
  'reference' =>
  string(40) "1530ed8512f5bf10f21c7f78f06eb43aa4760d0d"
  'name' =>
  string(28) "private-packagist/api-client"
}```

Reply:
It throws an error because it an error because you are running the code in the project and not when this package is installed. As you are currently on the master branch it does return `dev-master` as version.